src/courses/mappers/04.md

```diff
@@ -57,7 +57,7 @@ Also note that mappers make translating between two objects trivial, which means
 
 ## What Is an OHDF Mapper?
 
-An OHDF mapper is a mapper that allows the underlying conversion infrastructure found in OHDF Converters to correlate certain objects or values from one overarching security tool data format to another overarching security tool data format. In the case of the SAF tool suite, these mappers allow for the conversion of any given data format to OHDF (\*-to-OHDF) and vice versa (OHDF-to-\*) using the tools provided by the existing conversion infrastructure.
+An OHDF mapper is a mapper specifically focused on transforming security tool data formats to and from OHDF. It consists of a mapping and a number of helper functions which facilitate the actual application of the mapping. In the case of the SAF tool suite, these mappers allow for the conversion of any given data format to OHDF (\*-to-OHDF) and vice versa (OHDF-to-\*) using the tools provided by the existing conversion infrastructure.
```

Comment:
it doesn't need to be security tool data necessarily, just (cyber) security data in general.  for example, disa's checklist format isn't a security tool data format tho it is often used by security tools.

Reply:
Addressed.